_scala_items/7-education.md

```diff
@@ -0,0 +1,83 @@
+---
+shortTitle: "Ideal for teaching"
```

Comment:
The nature of this item (a use case of the language) is very different from the others items (language features). That does not look good to me.

I would not recommend to merge it, except if we have a clear idea on what other items we want and what is the timeline to make these changes. But this probably needs some thinking and discussion. Personally I think we should keep promoting some language features in the front page. So I would move "Scala for teaching" somewhere else.

Reply:
We will update the other items at some point. Hopefully before the end of the summer. The goal of this section is to give highlights on why we think Scala is interesting. So far, that section focused on language features, but we would like to make it more use-case oriented.